.appveyor.yml

```diff
@@ -64,11 +64,12 @@ artifacts:
 on_finish:
   - del C:\projects\devhub.key
   - codecov --disable=gcov
+  - ps: $blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```

Comment:
This needs to be removed or else it will start an Remote Desktop Session and block for quite some time (I think 1 hour). 

```
View report at: https://codecov.io/github/forcedotcom/salesforcedx-vscode/commit/28bb763e6220067c3152576f17e5ce5afa2d1559
$blockRdp = $true; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
Remote Desktop connection details:
  Server: 67.225.139.144:34414
  Username: appveyor
Build paused. To resume it, open a RDP session to delete 'Delete me to continue build.txt' file on Desktop.
```